.github/integration/tests/tests.sh

```diff
@@ -47,7 +50,7 @@ check_encypted_file $files
 
 # Upload a specific file and check it
 ./sda-cli upload -config testing/s3cmd.conf data_file.c4gh
-check_uploaded_file test/dummy/data_file.c4gh data_file.c4gh
+check_uploaded_file test/"$user"/data_file.c4gh data_file.c4gh
 
 output=$(./sda-cli list -config testing/s3cmd.conf | grep -q "data_file.c4gh")
```

Comment:
Note that `"$output"` will _always_ be empty since `grep -q` never outputs anything.

What I think is meant is this:
```
if ./sda-cli list -config testing/s3cmd.conf | grep -q 'data_file\.c4gh'
then
    echo 'Listed file from S3 backend'
else
    echo 'Failed to list file from S3 backend'
    exit 1
fi
```

Reply:
Indeed, good catch!